kafka_scripts/payment_status_stream.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE STREAM "mc_status" (CORRELATION_ID STRING, ERROR_CODE STRING)
```

Comment:
I think having a `kafka_scripts` directory and also a `scripts` directory is confusing - I propose that we change the `kafka_scripts` directory to just `kafka`, and make it a subdirectory of `scripts`.  I think we should do that via a new PR and GitHub issue, rather than amending it as part of this PR.  Thoughts, @cfagaras?

Reply:
A new issue is open for this topic and will be subject for another PR.
https://github.com/bb-mvp/kafka-pipeline/issues/39#issue-1537922188